src/App.css

```diff
@@ -87,3 +87,21 @@ li {
   border-bottom: 1px solid #e9e9e9;
   padding: 15px;
 }
+
+/* Missions */
+.wrapper {
+  width: 60rem;
+  margin: 15px auto;
+}
+
+.missions-bar,
+.missions-data {
+  display: grid;
+  grid-template-columns: 150px 500px 200px 150px;
+}
+
+.missions-bar div,
+.missions-data div {
+  border: 1px solid #e9e9e9;
+  padding: 10px;
+}
```

Comment:
- [ ]  Try to work with the alignment on the wireframe, center the button horizontally and vertically as it is on the wireframe design 

<br />

**Your interface**

![image](https://user-images.githubusercontent.com/58771507/156214690-1c786fb6-5b44-4dea-ac99-8f9e95cb5159.png)
<br />
**WireFrame**

![image](https://user-images.githubusercontent.com/58771507/156215147-1cc10117-8ad2-473a-b57f-617a0d5ac7d9.png)